src/sdk.ts

```diff
@@ -1352,6 +1362,76 @@ export default class SDK {
     return this.messenger.unlisten(this.identifier, handle);
   }
 
+  /**
+   * Transaction System
+   */
+
+  /**
+   * Calls for a list of products containing a sku, displayName, and cost.
+   *
+   * @async
+   * @since 2.4.0
+   *
+   * @throws {SDKError} Will throw an error if the MuxySDK didn't load.
+   * 
+   * @return {Promise<[]Product>} Resolves with an array of {@link Product}
+   * objects for each available sku.
+   * 
+   * @example
+   * const productList = sdk.getProducts();
```

Comment:
Function is galled `products` (should it be `getProducts`?), and returns a Promise. Should be: `const products = await sdk.products()`